lib/std/math.zig

```diff
@@ -887,6 +922,19 @@ fn testRem() !void {
     try testing.expectError(error.DivisionByZero, rem(f32, 10, 0));
 }
 
+/// Returns the absolute value of a floating point number.
+/// Uses a dedicated hardware instruction when available.
+/// This is the same as calling the builtin @fabs
+pub inline fn fabs(value: anytype) @TypeOf(value) {
+    return @fabs(value);
+}
+
+/// Alias to fabs for backwards compatibility
+/// This is the same as calling the builtin @fabs
+pub inline fn absFloat(value: anytype) @TypeOf(value) {
+    return @fabs(value);
+}
+
```

Comment:
We don't need to carry around backward compatibility cruft pre 1.0. This function is redundant and should be deleted.